lms/djangoapps/onboarding/models.py

```diff
@@ -253,6 +253,13 @@ def admin_info(self):
         """
         return "%s" % self.admin.email if self.admin else "Administrator not assigned yet."
 
+    def get_first_learner(self):
+        """
+        :return: User Object of  first learner of organization
+        """
+        first_learner_of_org = UserExtendedProfile.objects.filter(organization=self, is_first_learner=True).first()
```

Comment:
How about writing the query in this way and also renaming the variable to `org_first_learner` to simplify

```
org_first_learner = self.extended_profile.filter(is_first_learner=True).first()
if org_first_learner:
    return org_first_learner.user
```

Reply:
Also I have added this in **Organization** model  and org does not have the extended_profile property.